lib/spack/docs/packaging_guide.rst

```diff
@@ -4308,8 +4308,8 @@ follows:
                return
            # This implementation is lazy and only checks the first candidate
            exe_path = candidates[0]
-           exe = spack.util.executable.Executable(exe_path)
-           output = exe('--version')
+           exe = Executable(exe_path)
+           output = exe('--version', output=str, error=str)
```

Comment:
As is, this would never work. We almost always want to add `output=str` and `error=str` if we want to parse the output later.